src/renderer/Generics/redesign/Header/Header.tsx

```diff
@@ -109,14 +124,33 @@ export const Header = (props: NodeOverviewProps) => {
             )}
           </div>
         )}
-        <Button
-          {...startStopButtonProps}
-          variant="icon-left"
-          size="small"
-          type={
-            startStopButtonProps.iconId === 'play' ? 'primary' : 'secondary'
-          }
-        />
+        {/* In case of the status being an error, we should show both start & stop buttons so the user can try
+        both actions in the attempt to get the node working again. */}
+        {status.error ? (
+          <>
+            <Button
+              {...startButtonProps}
+              variant="icon-left"
+              size="small"
+              type="primary"
+            />
+            <Button
+              {...stopButtonProps}
+              variant="icon-left"
+              size="small"
+              type="secondary"
+            />
+          </>
+        ) : (
+          <Button
```

Comment:
on the design, it looks like start/stop buttons should only show up on the multiple node view, and not single node view.

if we have farcaster as a single node, would we need to differentiate whether the client is part of 2 node package, or single node package, and enable/disable the start/stop buttons based on that? 🤔

Reply:
some nodes will have optional clients ("services" in this case). so start stop at the single node view is necessary for these and if a user wants to change settings for a client without interfering with the other